ProteoformSuiteInternal/DeltaMassPeak.cs

```diff
@@ -30,12 +29,14 @@ public DeltaMassPeak(ProteoformRelation base_relation, List<ProteoformRelation>
             }
 
             grouped_relations = this.find_nearby_relations(relations_to_group);
-            this.peak_accepted = grouped_relations != null && grouped_relations.Count > 0 &&
-                typeof(TheoreticalProteoform).IsAssignableFrom(grouped_relations.First().connected_proteoforms[1].GetType()) ?
-                     this.peak_relation_group_count >= Lollipop.min_peak_count_et :
-                     this.peak_relation_group_count >= Lollipop.min_peak_count_ee;
-            this.possiblePeakAssignments = nearestPTMs(this.peak_deltaM_average);
-        }
+            this.peak_accepted = grouped_relations != null && grouped_relations.Count > 0 && grouped_relations.First().relation_type == ProteoformComparison.ExperimentalTheoretical ?
+                this.peak_relation_group_count >= Lollipop.min_peak_count_et :
+                this.peak_relation_group_count >= Lollipop.min_peak_count_ee;
+
+            possiblePeakAssignments = nearestPTMs(peak_deltaM_average, relation_type).ToList();
+            possiblePeakAssignments_string = "[" + String.Join("][", possiblePeakAssignments.Select(ptmset => String.Join(";", ptmset.ptm_combination.Select(m => m.modification.id)))) + "]";
+       }
```

Comment:
The intent of this display is to show all the possibilities, since the assignment for each relation is determined based on more information. You can separate it in notepad++ and other text editors by inserting a newline between braces. An example for why this is useful: missing isoleucine + oxidation has the same mass as missing proline. We should display both possibilities for the peak.